scripts/lib-franklin.js

```diff
@@ -577,18 +577,6 @@ export function loadHeader(header) {
   }
 }
 
-/**
```

Comment:
Do we need to remove it from `lib-franklin.js`? Also, can we touch this file? 🤔

Reply:
No we shouldn't change this file. I left the function as I, using the one from script.js